rules_building_block/credential_access_gdb_memory_dump.toml

```diff
@@ -0,0 +1,54 @@
+[metadata]
+creation_date = "2023/08/30"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/08/30"
+bypass_bbr_timing = true
+
+[rule]
+author = ["Elastic"]
+building_block_type = "default"
+description = """
+This rule monitors for potential memory dumping through gdb. Attackers may leverage memory dumping techniques to attempt
+secret extraction from privileged processes. Tools that display this behavior include "truffleproc" and 
+"bash-memory-dump". This behavior should not happen by default, and should be investigated thoroughly.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Linux Secret Dumping via GDB"
+references = [
+    "https://github.com/controlplaneio/truffleproc",
+    "https://github.com/hajzer/bash-memory-dump"
+]
+risk_score = 21
+rule_id = "66c058f3-99f4-4d18-952b-43348f2577a0"
+severity = "low"
+tags = ["Domain: Endpoint", "OS: Linux", "Use Case: Threat Detection", "Tactic: Credential Access", "Data Source: Elastic Defend", "Rule Type: BBR"]
+timestamp_override = "event.ingested"
+type = "eql"
+query = '''
+process where host.os.type == "linux" and event.action == "exec" and event.type == "start" and 
+process.name == "gdb" and process.args == "--pid"
```

Comment:
This one will duplicate the other rule alerts, do you think we should add an exclusion?

Reply:
Done